### DISCUSSION TITLE
Block All IP's Except allowed list

### DISCUSSION THREAD

**Discussion by rizdem** [AzuraCast/AzuraCast#5776] [Ideas]
Have a setting to auto block all ips from listening to streams unless they are on the allowed list.

I use CentovaCast for work which has the ability. but i would like to move away from this and host it myself. AzureCast is amazing it just misses this 1 feature :(

**Comment by BusterNeece** [comment]
I've just pushed an update to our Rolling Release channel that changes how the "Allowed IPs" list is processed; now, if you have any IPs in that list, it will be used as the authoritative list for what IPs are allowed to connect. If an IP isn't on the list, it will be considered blocked.

**Reply by rizdem** [reply]
Perfect.

All I can say is this is now flawless

**Reply by rizdem** [reply]
I have run the docker update as I used the one Click setup on linode says my version is  v0.17.3 Stable. 

Guessing this isn't the rolling release channel?

**Reply by BusterNeece** [reply]
Yes, by default you're on the Stable channel. To switch, see this guide: https://docs.azuracast.com/en/getting-started/updates/release-channels

**Reply by rizdem** [reply]
ahh such a noob thing. I've now got it working. Thank you

**Reply by rizdem** [reply]
@BusterNeece Looks like this is working on the first station I have setup using port 8000. any other stations setup after this point doesn't act in the same way and allows all ips even if I have an allowed list setup for that station

**Reply by BusterNeece** [reply]
@rizdem Figured this out and fixed it in an updated commit.
